async-nats/src/jetstream/kv/mod.rs

```diff
@@ -543,7 +543,7 @@ impl Store {
         Ok(())
     }
 
-    /// Purges the given key, destructively removing everything from the bucket.
+    /// Purges the given key history, destructively removing all key entries from the bucket.
```

Comment:
What about something ala

```suggestion
    /// Purges all the revisions of a key destructively, leaving a single purge marker in-place.
```

Reply:
Sure will update